### PR TITLE
feat(analytics): add events "more info" and download boarding pass

### PR DESCRIPTION
**Pull request by jalbertsr**
references https://github.com/kiwicom/smart-faq/issues/637
<br/><url>LiveURL: https://smartfaq-add-more-analytics-events.surge.sh</url>

### CODE REVIEW

**Comment by jaroslav-kubicek** [comment]
is this related to some issue? :)

File: src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js [hunk 1]
```diff
@@ -31,20 +42,34 @@ const BoardingPassesDescription = ({ data, mmbUrl }: Props) => {
         </p>
         <div className="info">
           {boardingPassUrl ? (
-            <a href={boardingPassUrl} target="_blank" className="download">
-              <Download size="medium" customColor="#00a991" />
-              Download
-            </a>
+            <div
+              onClick={() => downloadBoardingPassTracker()}
```
Comment (jaroslav-kubicek) [inline src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js]: wouldn't be just passing `downloadBoardingPassTracker` enough?

Comment (jaroslav-kubicek) [reply src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js]: also why wrapping by additional div, `<a onClick={...} />` should be IMHO enough

**Comment by jalbertsr** [comment]
@jaroslav-kubicek yes, sorry, I had a short meeting with Natali (she's in BCN summer office) and requested me some things, I will open an issue with a description for this PR and for more trackings we need.

File: src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js (hunk 1 above)
Comment (jalbertsr) [reply src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/BoardingPassesDescription.js]: double facepalm 🤦‍♂️, don't know what I was thinking 🙏, thanks

**Comment by jaroslav-kubicek** [comment]
ahh, of course, thanks!

**Pull request merged**